src/runtime/action/index.ts

```diff
@@ -17,9 +25,10 @@
  *
  * Docs: https://svelte.dev/docs#template-syntax-element-directives-use-action
  */
-export interface ActionReturn<Parameter = any> {
+export interface ActionReturn<Parameter = any, Attributes extends Record<string, any> = Record<never, any>> {
 	update?: (parameter: Parameter) => void;
 	destroy?: () => void;
+	$$attributes?: Attributes;
```

Comment:
I made this `$$` because that's what marks other things in the Svelte repo as "private" or "special". @Conduitry does this make sense?

Reply:
Yeah, it is convenient to use a single prefix for "reserved" - whether it's something that's special or private - but it can be confusing. What do you think about `$$_attributes` as a compromise?

Reply:
Sounds good, will do that together with a comment.